Dockerfile.local

```diff
@@ -33,14 +33,14 @@ RUN apk --no-cache upgrade && \
   apk add --update nodejs npm yq
 WORKDIR /app
 
-COPY --from=builder /app/core-ui/build /app/core-ui
```

Comment:
As we have a talk, please open new task as follow-up to change all `core-ui` from images etc